src/components/character-avatar/character-avatar.tsx

```diff
@@ -38,6 +42,19 @@ export const CharacterAvatar: FC<Props> = ({
   const [play, setPlay] = useState(false);
 
   const playSfx = usePlaySFX();
+  const isVotingDisabled = () => {
```

Comment:
Isn't this the same as:
`const isVotingDisabled = !isVoteEnabled || isLoadingVotes;`

What does `isVoteEnabled` signal? The naming of the inverted constants it's pretty hard to follow for a human.
Let's try to simplify this logic and clarify the naming.